src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -215,6 +215,28 @@ def makeLFNPrefixes(task):
 
     return temp_dest, dest
 
+def validateLFNs(path, files):
+    """
+    validate against standard Lexicon the LFN's that this task will try to publish in DBS
+    :param path: string: the path as created by CRAB starting from task name and possibly user LNFbase
+    :param files: list of strings: the filenames that will be published
+    :return: nothing if all OK. Raise assert exception if failure (as Lexicon functions do)
+    This is meant to be used early in the task preparation,
+    when the path and files passed do not have yet the directory counter and job_id counter,
+    so will insert take ones here when doing the check to ensure LFN lenght is correct
```

Comment:
Please fix the comment, I can understand it but that's after looking at the code.